ArduCopter/ekf_check.cpp

```diff
@@ -119,12 +119,23 @@ bool Copter::ekf_over_threshold()
     }
 
     // use EKF to get variance
-    float position_variance, vel_variance, height_variance, tas_variance;
+    float position_var, vel_var, height_var, tas_variance;
```

Comment:
I think the update to the filtered variances will need to be moved up above the ekf_thresh parameter check or else a user who disables the ekf failsafe will also disable the vibration failsafe.  Maybe we should add a comment as well to explain that the filtered variances are also used by check_vibration().

Reply:
Fixed